### PR TITLE
syncOperatorStatus: Simplify relatedObjects

### PR DESCRIPTION
**Pull request by miheer**
Simplify relatedObjects computation by omitting resource names.

https://jira.coreos.com/browse/NE-218

### CODE REVIEW

File: pkg/operator/controller/status.go [hunk 1]
```diff
@@ -71,14 +71,12 @@ func (r *reconciler) syncOperatorStatus() error {
...
-	for _, dns := range dnses {
-		related = append(related, configv1.ObjectReference{
+		{
 			Group:    operatorv1.GroupName,
 			Resource: "DNS",
-			Name:     dns.Name,
-		})
+		},
 	}
+
```
Comment (Miciah) [inline pkg/operator/controller/status.go]: The blank line can be deleted.

**Comment by ironcladlou** [comment]
Looks good, thanks!

/approve

**Comment by ironcladlou** [comment]
@Miciah will let you tag since you had a nit

File: pkg/operator/controller/status.go (hunk 1 above)
Comment (miheer) [reply pkg/operator/controller/status.go]: Done @Miciah  Thanks!

**Comment by Miciah** [comment]
Provisioning failure.
/test e2e-aws-upgrade

**Pull request merged**